msal/src/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -19,14 +31,12 @@
     <Summary>This package contains the binaries of the Microsoft Authentication Library (MSAL). MSAL makes it easy to obtain tokens from Azure AD v2 (work &amp; school accounts, MSA) and Azure AD B2C, gaining access to Microsoft Cloud API and any other API secured by Microsoft identities. This version supports adding authentication functionality to your .NET based client on Windows desktop (.NET 4.5+), UWP, .NET Core, Xamarin iOS and Xamarin Android.</Summary>
     <Copyright>(C) Microsoft Corporation. All rights reserved.</Copyright>
     <PackageTags>Microsoft Authentication Library;MSA;MSAL;B2C;Azure Active Directory;AAD;Identity;Authentication;.NET;Windows Store;Xamarin iOS;Android</PackageTags>
-    <Product>Microsoft.Identity.Client ($(TargetFramework))</Product>
```

Comment:
Don't use $(targetframework) here because it will just be filed with the last TFM from the list